.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": [ "@babel/preset-react", "@babel/preset-env" ]
+}
```

Comment:
I know it's just one of your config files, but consider adding a newline here as well.